Firebase/Core/Private/FIRAppInternal.h

```diff
@@ -109,42 +109,13 @@ extern NSString *const FIRAuthStateDidChangeInternalNotificationAppKey;
  */
 extern NSString *const FIRAuthStateDidChangeInternalNotificationUIDKey;
```

Comment:
Do we also want to move these constants to FIRAuthInterop?

Reply:
Hmm, I think that makes sense. We'll just need to make sure everyone using it changes at the same time. Oddly it looks like only Firestore uses this particular constant.

Reply:
Sounds good. I'll make another PR to switch everyone at once.